testing/src/main/java/com/linecorp/decaton/testing/TestUtils.java

```diff
@@ -137,14 +139,16 @@ private static Properties defaultProducerProps(String bootstrapServers) {
      * @param processorsBuilder actual processing logic. This parameter is mandatory
      * @param retryConfig can be null if you don't enable retry queueing feature
      * @param propertySupplier can be null if you don't supply extra Decaton configs
+     * @param stateListener can be null if you don't need to listen subscription state
      * @param <T> type of tasks
      * @return {@link ProcessorSubscription} instance which is already running
      */
     public static <T> ProcessorSubscription subscription(String subscriptionId,
                                                          String bootstrapServers,
                                                          ProcessorsBuilder<T> processorsBuilder,
                                                          RetryConfig retryConfig,
-                                                         PropertySupplier propertySupplier) {
+                                                         PropertySupplier propertySupplier,
+                                                         SubscriptionStateListener stateListener) {
```

Comment:
Umm.. feel like it start getting pointless and not usable to have this many of arguments in one method... 
We might have to change style passing builder to test code to let them do arbitrary thing in near future?

Reply:
Ya I'm thinking that too.

Cleaned up like this: https://github.com/line/decaton/pull/48/commits/4d269e8010873de93fc185ef2bfcb1567562ff7b